pages/code-of-conduct.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Code of Conduct"
+permalink: "/code-of-conduct/index.html"
+layout: "layouts/code-of-conduct.njk"
+tags:
+  - media
+---
+
+## La Murga de Austin Code of Conduct / Código de Conducta de La Murga de Austin
```

Comment:
I think this would read easier if we went ahead and consolidated the english text into one section and spanish in another. Instead of having the content go back and forth between language. 

The longer term fix would be to invest in some i18n content architecture: https://github.com/La-Murga-de-Austin/lmda-eleventy-netlify-cms/issues/8